### ISSUE TITLE
Drag and Drop layout vs grid.send layout?

### ISSUE THREAD

**Issue by JetsonDavis** [haltu/muuri#294]
I am getting a different layout when I manually drag and drop items from one grid to another versus when I send them.  I would like them to be the same - horizontal.  Can anyone advise on what is happening here:

![dnd mov](https://user-images.githubusercontent.com/875966/59811596-77ede900-92bf-11e9-90da-5f0c279a643b.gif)

 vs

![image](https://user-images.githubusercontent.com/875966/59811609-8cca7c80-92bf-11e9-9344-cb7f6ad9454d.png)

Many thanks!

**Comment by niklasramo** [comment]
Weird behaviour indeed. I have good guess what might cause those behaviours, but a live demo would be helpful here so one can tinker with the code.

**Comment by JetsonDavis** [comment]
Yep, I'll whip something up tmrw - getting late here in California!  Thanks Niklas!

**Comment by JetsonDavis** [comment]
Here's a PEN I created which duplicates the error.  I tried to add (many of) the libraries I am using.  I also tried to make it look as much like my app as possible.  You will see the same issues as above if you drop n drag vs. pushing the arrow button to execute a 'grid.send'.

https://codepen.io/JetsonDavis/pen/LKxQEB

Thanks!

**Comment by JetsonDavis** [comment]
I'm a JavaScript novice but if you can suggest places to look, I can try to help debug the code!?
